app/assets/stylesheets/application.css.scss

```diff
@@ -25,10 +25,20 @@
   padding-top: 50px;
 }
 
-body {
+.picture-container{
   background-image: url('http://wallpaperpassion.com/upload/13897/yoga-wallpaper.jpg');
-  background-repeat:no-repeat;
-  background-size: 100% 100%;
+  // background-repeat:no-repeat;
+  // background-size: 100% 100%;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background-image, background-position, -moz-background-size, -o-background-size, -webkit-background-size, background-size, height, position